src/Sulu/Bundle/AudienceTargetingBundle/Tests/Unit/Controller/TargetGroupEvaluationControllerTest.php

```diff
@@ -80,31 +82,27 @@ public function testTargetGroupAction($header, $currentTargetGroup, $targetGroup
         $this->assertEquals($targetGroupId, $response->headers->get($header));
     }
 
-    public function provideTargetGroup()
+    public static function provideTargetGroup(): array
     {
-        $targetGroup1 = $this->prophesize(TargetGroupInterface::class);
-
-        $targetGroup2 = $this->prophesize(TargetGroupInterface::class);
-        $targetGroup2->getId()->willReturn(2);
-
-        $targetGroup3 = $this->prophesize(TargetGroupInterface::class);
-        $targetGroup3->getId()->willReturn(3);
-        $targetGroup4 = $this->prophesize(TargetGroupInterface::class);
-        $targetGroup4->getId()->willReturn(4);
+        $targetGroup1 = self::createStub(TargetGroupInterface::class);
```

Comment:
we will keep using prophesize currently for mocks we not using PHPUnit Stub or Mockbuilder